conda/models/match_spec.py

```diff
@@ -778,11 +778,14 @@ def match(self, other):
         if self._re_match:
             return self._re_match(_other_val.canonical_name)
         else:
-            return self._raw_value.canonical_name == _other_val.canonical_name
+            # assert ChannelMatch('pkgs/free').match('defaults') is False
+            # assert ChannelMatch('defaults').match('pkgs/free') is True
```

Comment:
Do you want to retain these comments?

Reply:
Yeah, I do.  Just to help clarify in this section of code the order of channel <=> multi-channel matching.